src/app/components/VisualImage.js

```diff
@@ -3,18 +3,21 @@ function VisualImage(imageSource) {
   // FIXME: If we should use only base64/jpg, delete one condition in if
   const reg = /^data:image\/(?:gif|png|jpeg|bmp|webp)(?:;charset=utf-8)?;base64,(?:[A-Za-z0-9]|[+/])+={0,2}/;
   if (reg.test(imageSource) || /.jpg/.test(imageSource)) {
-    // Creating div and img
-    const createDiv = document.createElement('div');
-    createDiv.className = 'visual-box';
+    // check if image div is existing
+    const isDivExisting = document.querySelector('.visual-box');
+    if (!isDivExisting) {
+      // Creating div and img
+      const createDiv = document.createElement('div');
+      createDiv.className = 'visual-box';
 
-    const createImg = document.createElement('img');
-    createImg.classList.add('visual-box__image');
-    createImg.setAttribute('alt', 'image');
-
-    // Placing div with img inside container with id = "swquiz-app"
-    document.querySelector('#swquiz-app').appendChild(createDiv);
-    document.querySelector('.visual-box').appendChild(createImg);
+      const createImg = document.createElement('img');
+      createImg.classList.add('visual-box__image');
+      createImg.setAttribute('alt', 'image');
 
+      // Placing div with img inside container with id = "swquiz-app"
+      document.querySelector('#swquiz-app').appendChild(createDiv);
+      document.querySelector('.visual-box').appendChild(createImg);
+    }
     // Adding sourse to img
     const img = document.querySelector('.visual-box__image');
     img.setAttribute('src', imageSource);
```

Comment:
would be good to return the div with image, so we could insert it anywhere it was needed